README.md

```diff
@@ -26,6 +26,15 @@ Testing on IE11 requires a bit more setup.
 
 Update dependency in package.json to e.g. `"posthog-js": "link:../posthog-js"`, `yarn` and run `yarn build && yarn build-module`
 
+An alternative is to use [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link/). Run `yarn link` in `posthog-js`, and then `yarn link posthog-js` in `posthog`. Once you're done, remember to `yarn unlink`.
+
+### Developing with main PostHog repo
+
+The `posthog-js` snippet for a website loads static js from the main `PostHog/posthog` repo. Which means, when testing with a website running the snippet, you want to run `PostHog/posthog` locally, link the `posthog-js` dependency to your local version, and then run `yarn serve` in `posthog-js` to ensure `dist/array.js` is being generated.
+
+Then, in your locally running `PostHog/posthog` build, run `yarn copy-scripts` so the scripts copied from `posthog-js` are served via `PostHog/posthog`. Further, it's a good idea to modify `start-http` script to add development mode: `webpack serve --mode development`, which doesn't minify the resulting js (which you can then read in your browser).
+
```

Comment:
I'm all in for a block like this, but this one reads way too complicated right now. It says everything and it's hard to parse out the relevant information.

Can you rephrase it in the style of "if this is what you want, this is what you do". E.g. "If you're developing posthog locally and want it to run with a local copy of posthog-js, run these 4 commands you can copy/paste."... 

Later we can expand to "if you want to run a dev version of posthog-js on your own saas app, you can do this".

Reply:
Sorry, I was being lazy.

Makes sense - will do!